### DISCUSSION TITLE
feedback Fritz!Repeater 450E

### DISCUSSION THREAD

**Discussion by baetzst** [Freetz-NG/freetz-ng#455] [Allgemein]
Hi,
da noch untestet dabei steht hier mein feedback.
Ich habe freetz-ng für meinen 450E mit dnsmasq, rsync und dropbear gebaut und nutze diesen als DHCP Server.
Läuft seit 3 Tagen ohne Probleme.

**Comment by fda77** [comment]
Danke! Kannst du noch einen Screenshot der "AVM-Info" Seite Anhängen? Nur dort sieht man genau die CPU
 -> https://github.com/Freetz-NG/freetz-ng/commit/c9370eaa9bb6736950a87454bcbac832a1ff41c9

**Comment by baetzst** [comment]
logisch
![Unbenannt](https://user-images.githubusercontent.com/77538048/151698652-b6393a64-ea0e-4317-af78-80e02660066c.PNG)
